### DISCUSSION TITLE
When the results from the paper and FDS differ

### DISCUSSION THREAD

**Discussion by MNJOLE** [firemodels/fds#12872] [Q&A]
Hello, I am currently practicing implementing a paper based on reference and comparing it with the results of FDS. I have simulated the properties and implementation method exactly as described in the paper. However, the results of FDS implementation are very different from the paper. I would really appreciate it if you could suggest what I could modify or guess which part of the code might be problematic.
![cone heater](https://github.com/firemodels/fds/assets/166811458/d2bb8b07-8268-413a-ae21-8067fa46d2c0)
![HRR_1](https://github.com/firemodels/fds/assets/166811458/ffea94eb-1290-46a6-936b-bd8ac76351a1)
![PMMA surface temperature](https://github.com/firemodels/fds/assets/166811458/ce140f29-ba45-4321-9b89-93618f7d1207)
![surf temperature](https://github.com/firemodels/fds/assets/166811458/b6eee68b-89c5-41b3-ac12-028f146a05d8)
![thermal properties MMA](https://github.com/firemodels/fds/assets/166811458/1ce4d76e-8ca1-4581-af8b-6639e9ef6537)
![thermal properties PMMA](https://github.com/firemodels/fds/assets/166811458/1827e480-35e7-46f3-8800-31aea3edccef)

The following is the information provided in the paper.
It uses DNS model and the chemical reaction equation is as follows,
![reaction](https://github.com/firemodels/fds/assets/166811458/9077ae7f-73fb-47d3-a4c9-002a274f7e30)

and below is the code that I have written.
&HEAD CHID='v16'/
&TIME T_END=800.0/
&MISC SIMULATION_MODE='DNS'/

&RAMP ID='MMA_RAMP_MU', T=98.85, F=9.44E-6/
&RAMP ID='MMA_RAMP_MU', T=100.85, F=1.088E-5/
&RAMP ID='MMA_RAMP_MU', T=126.85, F=1.179E-5/
&RAMP ID='MMA_RAMP_MU', T=326.85, F=1.903E-5/
&RAMP ID='MMA_RAMP_MU', T=526.85, F=2.667E-5/
&RAMP ID='MMA_RAMP_MU', T=626.85, F=3.065E-5/
&RAMP ID='MMA_RAMP_MU', T=727.85, F=2.45E-5/

&RAMP ID='MMA_RAMP_D', T=0.0, F=7.64E-6/
&RAMP ID='MMA_RAMP_D', T=13.85, F=7.64E-6/
&RAMP ID='MMA_RAMP_D', T=15.85, F=7.69E-6/
&RAMP ID='MMA_RAMP_D', T=126.85, F=1.35E-5/
&RAMP ID='MMA_RAMP_D', T=326.85, F=2.76E-5/
&RAMP ID='MMA_RAMP_D', T=526.85, F=4.57E-5/
&RAMP ID='MMA_RAMP_D', T=626.85, F=5.61E-5/
&RAMP ID='MMA_RAMP_D', T=727.85, F=1.37E-4/

&RAMP ID='MMA_RAMP_CP', T=98.85, F=0.015/
&RAMP ID='MMA_RAMP_CP', T=126.85, F=0.01692/
&RAMP ID='MMA_RAMP_CP', T=326.85, F=0.0319/
&RAMP ID='MMA_RAMP_CP', T=526.85, F=0.04836/
&RAMP ID='MMA_RAMP_CP', T=626.85, F=0.057145/
&RAMP ID='MMA_RAMP_CP', T=727.85, F=0.066/

&RAMP ID='MMA_RAMP_K', T=0.0, F=0.753/
&RAMP ID='MMA_RAMP_K', T=13.85, F=0.753/
&RAMP ID='MMA_RAMP_K', T=15.85, F=0.75669/
&RAMP ID='MMA_RAMP_K', T=126.85, F=1.055784/
&RAMP ID='MMA_RAMP_K', T=326.85, F=1.506596/ 
&RAMP ID='MMA_RAMP_K', T=526.85, F=1.844872/
&RAMP ID='MMA_RAMP_K', T=626.85, F=1.972086/
&RAMP ID='MMA_RAMP_K', T=727.85, F=2.072354/

&SPEC ID='MMA', FORMULA='C5H8O2', 
RAMP_D='MMA_RAMP_D', 
RAMP_MU='MMA_RAMP_MU', 
RAMP_CP='MMA_RAMP_CP',
RAMP_K='MMA_RAMP_K'/

&SPEC ID='OXYGEN', MASS_FRACTION_0=0.23/
&SPEC ID = 'CARBON DIOXIDE' /
&SPEC ID = 'WATER VAPOR' /

&REAC ID='SFPE PMMA',
      FYI='SFPE Handbook, 5th Edition, Tables A.38 and A.39',
      FUEL='MMA',
SPEC_ID_NU='MMA','OXYGEN','CARBON DIOXIDE','WATER VAPOR',
NU=-1,-6,5,4,
SPEC_ID_N_S='MMA', 'OXYGEN',
N_S=1.,1./

&MATL ID='PMMA',
      SPECIFIC_HEAT=1.69,
      CONDUCTIVITY=0.336,
      DENSITY=1155.0,
      ABSORPTION_COEFFICIENT=2240.0,
      N_REACTIONS=1,
      HEAT_OF_REACTION=1830.0,
      A=8.6E+12,
      E=1.881E+5,
      SPEC_ID=MMA,
      NU_SPEC=1/

&SURF ID='ADIABATIC',
      COLOR='GRAY 80',
      ADIABATIC=.TRUE./
&SURF ID='PMMA',
      RGB=26,204,26,
	BACKING='EXPOSED',
      BURN_AWAY=.TRUE.,
      MATL_ID(1,1)='PMMA',
      MATL_MASS_FRACTION(1,1)=1.0,
      THICKNESS(1)=0.01/
&SURF ID='vent',
      RGB=26,128,26,
      VOLUME_FLOW=0.024/
&SURF ID='heatflux',
      COLOR='RED',
      TMP_FRONT=774.0/

![image](https://github.com/firemodels/fds/assets/166811458/9c268bac-a78c-4777-b1ba-2e786ace2e7c)

**Comment by TristanHehnen** [comment]
The very first thing I would suggest is to make sure that you are using the same FDS version. Check the `CHID.out` file, the line with the revision, for example "Revision: FDS6.7.7-0-gfe0d4ef38-release". Also, state this information with the document/article/thesis you write.
Also, it may be helpful to reach out to the authors and talk to them how they implemented their ideas. Often not the the full information is provided in articles.

**Reply by MNJOLE** [reply]
Revision         : FDS6.7.5-0-g71f025606-release , 
and thank you for your advice. As a last resort, I'm considering reaching out to the authors directly.

**Comment by TristanHehnen** [comment]
Further questions would be how the cone heater is modeled (OBST, GEOM, EXTERNAL_FLUX, ...), which radiative fraction is used, was the radiation model adjusted, was the extinction model adjusted, are the same cell sizes used, and so on.

You may also want to check out this [discussion about MMA combustion](https://github.com/firemodels/fds/discussions/12599).

Could you please link the article you are discussing?

**Reply by MNJOLE** [reply]
I created a thin rectangular prism in Pyrosim and applied TMP_FRONT=774.0 to one side. Then, I rotated it to create a conical heater shape.
The paper is  Korean master's thesis, so I'm not sure if this is okay, but I'll attach it,
https://oak.chosun.ac.kr/bitstream/2020.oak/13545/2/%ec%bd%98%ec%b9%bc%eb%a1%9c%eb%a6%ac%eb%af%b8%ed%84%b0%eb%a5%bc%20%ec%9d%b4%ec%9a%a9%ed%95%9c%20%eb%b9%84%20%ed%83%84%ed%99%94%20%ea%b3%a0%ec%b2%b4%20%ea%b0%80%ec%97%b0%eb%ac%bc%ec%9d%98%20%ec%97%b4%eb%b6%84%ed%95%b4%20%eb%b0%8f%20%ec%97%b0%ec%86%8c%20%ed%8a%b9%ec%84%b1%ec%97%90%20%ea%b4%80%ed%95%9c%20%ec%97%b0%ea%b5%ac.pdf

**Reply by TristanHehnen** [reply]
Thank you.

Not sure if I understand it correctly, from figure 4.4 it looks like they may have used FDS 6.0.


So it seems that they used OBST for the heater as well. 

From figure 4.15 the heat flux map looks rather smooth, I would have expected more variation, specifically lower values in the corners.
I can't speak Korean but I've got the impression that they discuss in the section below said figure that the heat flux may be lower than they would like.
I did write a Python script that can calibrate the TMP_FRONT such that the gauge heat flux at a DEVC shows the desired heat flux, mimicking the procedure used with the real cone. Not sure if this helps you, but since the FDS version may be rather old you may redo things anyway. 
You can find the script in the dataset provided with our [article](https://doi.org/10.1016/j.firesaf.2023.103926).
There is also a cone heater GEOM in the dataset you could use if you like.

All of this may not solve your issue though...

**Reply by MNJOLE** [reply]
There were various methods to implement a cone heater... I'll try this method as well. Thank you.

**Comment by drjfloyd** [comment]
You did not give your entire input file as there is no &MESH, &OBST, or &VENT lines. You can attach a .fds file to a discussions by changing the extension to .txt. For the lines you have given I see three things:
-You didn't define NITROGEN as the background species.
-You appear to have the RAMPs for k and cp switiched.  For gasses, k should be on the order of 0.01 and cp on the order of 1.  
-In a cone the PMMA sample will start to pyrolyze a short bit before the fire ignites. In FDS, with the default AUTO_IGNITION_TEMPERATURE as soon as the PMMA begins to pyrolyze it will burn providing additional heat to the surface. The paper results show a longer delay to the first HRR and the authors may have set an IGNITION_TEMPERATURE.

**Reply by drjfloyd** [reply]
Another issue just occured to me. Your SURF is simply PMMA backed by ambient air. This is not what was tested in the cone which was PMMA ontop of ceramic wool. Backed by air the sample will loose heat off the back. Backed by ceramic wool that heat will remain in the sample and result in higher rates of pyrolysis.

**Reply by MNJOLE** [reply]
Thank you for your advice. However, I have a few questions. 
1. When modeling the thermal decomposition of solid combustibles, is it always necessary to include nitrogen? 
2. To make the changes as you suggested, should I add an additional SURF for ceramic wool, or should I change the backing of PMMA to "insulated"?

I've attached the code I'm currently interpreting, where I changed ramp k and cp as you suggested. It seems to be a paper implementing FDS using a thermal decomposition model without specifying an ignition temperature.

[cone heater.txt](https://github.com/firemodels/fds/files/15195376/cone.heater.txt)

**Reply by drjfloyd** [reply]
1.   &SPEC ID='OXYGEN', MASS_FRACTION_0=0.23/   What is the other 77 % of the gas present in the domain? FDS normally assumes the background species is AIR but you are not using simple chemistry so you need to define the background species.
2.  INSULATED would put a perfect insulator at the back of the PMMA sample.  Is ceramic wool a perfect insulator (k=0)? If not, then treating it as a perfect insulator will be a source of uncertainty in your prediction.

**Reply by drjfloyd** [reply]
You have a 4 mm grid. This is much too large for DNS. 1 mm or less is needed.
With DNS, if you do not specify a radiant fraction for your fuel, FDS will assume you wish to predict the radiant fraction. You haven't specified any soot in your combustion reaction which is a significant species for radiation when it is present.
